src/Client.php

```diff
@@ -166,15 +180,32 @@ public function register(RabbitMqObjectInterface $object)
     }
 
     /**
+    * Function used to create $this->connection
+    * handle connection's retry in case of the rabbitmq server is not ready yet
+    **/
+    private function createConnection()
+    {
+        try {
+            $this->connection = new AMQPStreamConnection($this->host, $this->port, $this->user, $this->password);
+        } catch (\ErrorException $e) {
```

Comment:
I'm not completely comfortable catching a \ErrorException.
What is the error precisely thrown when a connexion problem occurs?